yogstation/code/game/machinery/computer/arcade.dm

```diff
@@ -0,0 +1,3 @@
+/obj/machinery/computer/arcade/Initialize()
+	 .=..()
+	 prizes[/obj/item/grenade/plastic/glitterbomb/pink] = 1
```

Comment:
I'm sorry but it can't be pink, due to our bigotry rule, we can't risk anyone seeing the pink colour which would make them go "damn that's gay".